### ISSUE TITLE
Please support having `in` natively for bytes

### ISSUE THREAD

**Issue by paulm17** [prisma/prisma#8854]
### Bug description

Following on from https://github.com/prisma/prisma/issues/8686.

Byte Ids should support using an array when using where in.

```
    const bufferIds = ids.map(id => {
      return Buffer.from(id)
    })

    const query = await db.damFile.updateMany({
      data: values,
      where: {
        id: {
          in: bufferIds,
        },
        userId: Buffer.from(ctx.session.userId),
      },
    })
```

I'm getting the following typescript error:

```
Type '{ in: Buffer[]; }' is not assignable to type 'Buffer | BytesFilter | undefined'.
  Object literal may only specify known properties, and 'in' does not exist in type 'Buffer | BytesFilter'.ts(2322)
```

Upon trying the query, I get the following:

```
	Invalid `prisma.damFile.updateMany()` invocation:

	{
	data: {
		description: 'wwwwwwgggggghhh'
	},
	where: {
		id: {
		in: [
		~~
			Buffer(3)
		]
		},
		userId: Buffer(3)
	}
	}

	Unknown arg `in` in where.id.in for type BytesFilter. Available args:

	type BytesFilter {
	equals?: Bytes
	not?: Bytes | NestedBytesFilter
	}
```

Many thanks.

### How to reproduce

Here is some data you can try:

01FB6VB2G907FMJ91P88VXGA4S
01FB6VB4XWA2YKJWVZ7KADQF56
01FB6VB6W349W5SHTNSE39674X
01FB6VB95R8QB2SXAX87GVC7KY
01FB6VBAYKAYG0Y80DKJ4SQXQN

### Expected behavior

No typescript error and the query should execute.

### Prisma information

```
model DamFile {
   id                Bytes     @id @db.Binary(26)
   description       String?   @db.LongText
   comments          String?   @db.LongText
   copyright         String?   @db.VarChar(191)
   publicationDate   DateTime?
}
```

### Environment & setup

- OS: OSX Mavericks
- Database: mysql Ver 15.1 Distrib 10.5.11-MariaDB, for osx10.14 (x86_64) using readline 5.1
- Node.js version: v14.15.4

### Prisma Version

```
Environment variables loaded from .env
prisma               : 2.29.0
@prisma/client       : 2.29.0
Current platform     : darwin
Query Engine         : query-engine 1be4cd60b89afa04b192acb1ef47758a39810f3a (at node_modules/prisma/node_modules/@prisma/engines/query-engine-darwin)
Migration Engine     : migration-engine-cli 1be4cd60b89afa04b192acb1ef47758a39810f3a (at node_modules/prisma/node_modules/@prisma/engines/migration-engine-darwin)
Introspection Engine : introspection-core 1be4cd60b89afa04b192acb1ef47758a39810f3a (at node_modules/prisma/node_modules/@prisma/engines/introspection-engine-darwin)
Format Binary        : prisma-fmt 1be4cd60b89afa04b192acb1ef47758a39810f3a (at node_modules/prisma/node_modules/@prisma/engines/prisma-fmt-darwin)
Default Engines Hash : 1be4cd60b89afa04b192acb1ef47758a39810f3a
Studio               : 0.419.0

```

**Comment by serejkaaa512** [comment]
Is there anything can be done with it? I have faced this issue having `Bytes` as pk and fk in another table to it. I use prisma + nexus + Apollo as a backend. The request of objects list with `Bytes` pk and fk objects inside it leads to error `Value types mismatch. Have: Object({\"in\": List([..])}), want: Bytes`. As I understand it happens because there is no `in` in BytesFilter.

**Comment by serejkaaa512** [comment]
As I understand ts library doesn't generate `in` filter for `Bytes`. [Here](https://github.com/prisma/prisma/blob/bbdf1c23653a77b0b5bf7d62efd243dcebea018b/packages/client/src/generation/TSClient/Input.ts#L89) is only check for 'number' and `Buffer` is not passing it. 
There are tests for these filters for `bytes` in Rust Prisma Engine and I think that this should work in generated TS client.
@janpio Can you take a look? Or am I wrong?

**Comment by serejkaaa512** [comment]
As I understand (this)[https://github.com/prisma/prisma-engines/blob/master/query-engine/core/src/schema_builder/input_types/field_filter_types.rs#L184] must be fixed. TypeIdentifier::Bytes must be separated and ` .chain(inclusion_filters(mapped_scalar_type.clone(), nullable))` is added to it.

**Comment by dpetrick** [comment]
This will be released with 3.6.
